README.md

```diff
@@ -19,6 +19,13 @@ $ source .env
 $ export $(cut -d= -f1 .env)
 ```
 
+FOr first time activation
```

Comment:
@StefanTrsunov please make the `O` character to be lowercase

Reply:
I did the change